docs/api/instruments/edwards/index.rst

```diff
@@ -0,0 +1,12 @@
+.. module:: pymeasure.instruments.edwards
+
+#####
+Edwards
+#####
+
```

Comment:
`#` chars should be same length of `Edwards` string.
Also file `pymeasure/docs/API/instruments/index.rst` should be updated to include `edwards` directory.

BTW: did you know you can  run `make html` from `docs` directory to check that no warning are present in documentation?

Reply:
How do I update `pymeasure/docs/API/instruments/index.rst`  to include edwards directory?